src/Sylius/Bundle/ApiBundle/Resources/config/integrations/swagger.xml

```diff
@@ -23,10 +23,10 @@
             <argument type="tagged_iterator" tag="sylius.open_api.modifier" />
         </service>
 
-<!--        <service id="Sylius\Bundle\ApiBundle\OpenApi\Documentation\AcceptLanguageHeaderDocumentationModifier">-->
-<!--            <argument type="service" id="sylius.repository.locale" />-->
-<!--            <tag name="sylius.open_api.modifier" />-->
-<!--        </service>-->
+        <service id="Sylius\Bundle\ApiBundle\OpenApi\Documentation\AcceptLanguageHeaderDocumentationModifier">
```

Comment:
dot notation? 🤔

Reply:
It should be, but not just for those enabled, but for the whole, so it could be done in the separate PR 🖖🏻